packages/frontend/src/hooks/useExplorerRoute.ts

```diff
@@ -411,6 +445,8 @@ export const useExplorerRoute = () => {
                         ...unsavedChartVersion,
                         metricQuery,
                     },
+                    undefined,
+                    expandedSections,
```

Comment:
this persists the expandedSections after url parsing on the url. 
I'm with it persisting and not persisting - the ideal result is for business users to not see the chart plotted on clicking ´explore´